backup/setup/dbimport.bat

```diff
@@ -0,0 +1,49 @@
+@echo off && setlocal
+
+rem set configuration variables
+SET DEFAULT_DB=schulcloud
+SET DEFAULT_HOST=localhost:27017
+SET CREDENTIALS=-u username -p pwd
```

Comment:
I need to remove this line, otherwise I got:
Failed: error connecting to db server: server returned error on SASL authentication step: Authentication failed.